### PR TITLE
[USING] explain bip 39

### PR DESCRIPTION
**Pull request by MaxHillebrand**
This ready for review branch explains bip 39 mnemonic words

### CODE REVIEW

File: docs/using-wasabi/BIPs.md [hunk 1]
```diff
@@ -38,7 +38,8 @@ Here is a list of all the supported and integrated Bitcoin Improvement Proposals
 :::details
 ### BIP 39: Mnemonic Code for Generating Deterministic Keys
 
-[BIP 39: Mnemonic Code for Generating Deterministic Keys](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki)
+[BIP 39: Mnemonic Code for Generating Deterministic Keys](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) is a method to display the backup of private keys in easy to read sentence of 12, 18 or 24 words.
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/BIPs.md:41]:
```suggestion
[BIP 39: Mnemonic Code for Generating Deterministic Keys](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) is a method to display the backup of private keys in easy to read sentence of 12, 18, 24 or more words.
```

Comment (MaxHillebrand) [reply docs/using-wasabi/BIPs.md:41]: I don't think there are more than 24 words in a mnemonic standard?

Comment (RiccardoMasutti) [reply docs/using-wasabi/BIPs.md:41]: You are right. I made this change because I don't want users to get confused with the seed + passphrase lenght.

Comment (MaxHillebrand) [reply docs/using-wasabi/BIPs.md:41]: yeah, but the 25th word is another BIP, I believe?

Comment (lontivero) [inline docs/using-wasabi/BIPs.md:41]: No. The bip39 is NOT a method to display the backup of private keys in easy to read sentence of 12, 18 or 24 words. It is a way to encode the seed used in the generation of deterministic wallets.

NACK. Why are we explaining the BIPs? They are really technical specifications.

**Pull request closed without merging**